.env.example

```diff
@@ -1,4 +1,3 @@
-NEXT_PUBLIC_UAT_URL=
-NEXT_PUBLIC_TEST_API=
+NEXT_PUBLIC_DESIGN_URL=
```

Comment:
Why it is called DESIGN URL? I thought it's for the home page(marketing page) URL.

Reply:
I think  this URL for the 404-page redirects to the design page.

Reply:
Thanks for reminding me